### ISSUE TITLE
sqlite3def Windows AMD64 binary is a stub, not working

### ISSUE THREAD

**Issue by shunf4** [k0kubun/sqldef#149]
## --export output

2021/09/06 11:46:33 Error on DumpDDLs: Binary was compiled with 'CGO_ENABLED=0', go-sqlite3 requires cgo to work. This is a stub

## Input SQL

(any)

## Current output

Under Windows, using sqlite3def_windows_amd64 :

```
> .\sqlite3def.exe /f sqlite3.sql test.db
2021/09/06 11:43:09 Error on DumpDDLs: Binary was compiled with 'CGO_ENABLED=0', go-sqlite3 requires cgo to work. This is a stub
```

## Expected output

```
> .\sqlite3def.exe /f sqlite3.sql test.db
(Something like applying statements, without error)
```

**Comment by k0kubun** [comment]
I haven't changed anything about it since https://github.com/k0kubun/sqldef/pull/64. @mattn Do you have any idea about this?

**Comment by mattn** [comment]
@shunf4 Do you install gcc compiler on your environment?

**Comment by shunf4** [comment]
> @shunf4 Do you install gcc compiler on your environment?

Yes.

```
> where gcc
C:\ProgramData\chocolatey\bin\gcc.exe
```

But I don't quite get you. Do I have to install MinGW gcc or something, just in order to use an executable binary?

Also, it's only the binary downloaded from [Release page](https://github.com/k0kubun/sqldef/releases) that is not working. if installed from `go install github.com/k0kubun/sqldef/cmd/sqlite3def@latest`, the binary works just fine.

**Comment by mattn** [comment]
Could you provide your `go env` ?

**Comment by shunf4** [comment]
> Could you provide your `go env` ?

```
set GO111MODULE=
set GOARCH=amd64
set GOBIN=
set GOCACHE=C:\Users\shunf4\AppData\Local\go-build
set GOENV=C:\Users\shunf4\AppData\Roaming\go\env
set GOEXE=.exe
set GOFLAGS=
set GOHOSTARCH=amd64
set GOHOSTOS=windows
set GOINSECURE=
set GOMODCACHE=C:\Users\shunf4\go\pkg\mod
set GONOPROXY=
set GONOSUMDB=
set GOOS=windows
set GOPATH=C:\Users\shunf4\go
set GOPRIVATE=
set GOPROXY=https://proxy.golang.org,direct
set GOROOT=C:\Program Files\Go
set GOSUMDB=sum.golang.org
set GOTMPDIR=
set GOTOOLDIR=C:\Program Files\Go\pkg\tool\windows_amd64
set GOVCS=
set GOVERSION=go1.16.6
set GCCGO=gccgo
set AR=ar
set CC=gcc
set CXX=g++
set CGO_ENABLED=1
set GOMOD=NUL
set CGO_CFLAGS=-g -O2
set CGO_CPPFLAGS=
set CGO_CXXFLAGS=-g -O2
set CGO_FFLAGS=-g -O2
set CGO_LDFLAGS=-g -O2
set PKG_CONFIG=pkg-config
set GOGCCFLAGS=-m64 -mthreads -fmessage-length=0 -fdebug-prefix-map=C:\Users\shunf4\AppData\Local\Temp\go-build2196836561=/tmp/go-build -gno-record-gcc-switches
```

**Comment by mattn** [comment]
Ah.

>Also, it's only the binary downloaded from Release page that is not working. 

`tcnksm/ghr` does not support Windows CGO build on Linux. To build Windows CGO binary, it need to use https://github.com/karalabe/xgo.

**Comment by shunf4** [comment]
So maybe remove Windows builds from Release page, and add some description on how to build it on Windows?

Does darwin builds have the same problem?

**Comment by mattn** [comment]
> So maybe remove Windows builds from Release page, and add some description on how to build it on Windows?

Yes, or switch to use xgo.

> Does darwin builds have the same problem?

Maybe so.

**Comment by k0kubun** [comment]
> Does darwin builds have the same problem?

```
$ otool -L sqlite3def
sqlite3def:
        /usr/lib/libSystem.B.dylib (compatibility version 0.0.0, current version 0.0.0)
$ ./sqlite3def -f /tmp/test.sqlite3 test < /tmp/a.sql
2021/09/26 13:35:06 Error on DumpDDLs: Binary was compiled with 'CGO_ENABLED=0', go-sqlite3 requires cgo to work. This is a stub
```

seems like it. I'll remove non-working binaries from the releases for now. I'll add a description about how to build it once I figure it out. Thank you for your report.

**Comment by k0kubun** [comment]
I addressed this issue in v0.12.8. The binary release should work on Windows as well.
